### ISSUE TITLE
Adopt as the new official "mozilla-gnome-keyring"

### ISSUE THREAD

**Issue by infinity0** [swick/mozilla-gnome-keyring#28]
Hi, after #27 is merged, I'd be happy to bless this as the next "official" version of mozilla-gnome-keyring, and hand over maintenance of this software to you. I actually already have a demo debian package of this [here](https://mentors.debian.net/package/mozilla-gnome-keyring).

I'd prefer it if the name was kept, though. It has some presence online and it would provide some continuity with the current mozilla-gnome-keyring. Are you OK with doing this? It would involve changing the extension URL on addons.mozilla.org, and updating the places in the repo to change "moz-gnome-keyring-integration" to "mozilla-gnome-keyring".

**Comment by infinity0** [comment]
Hey, any news on this?

**Comment by infinity0** [comment]
Hey, any news on this? I've uploaded this to [Debian experimental](https://packages.debian.org/experimental/xul-ext-gnome-keyring) but waiting for your approval to make it "official". There is still the issue of #27 which fixes a bug and adds an icon.

**Comment by swick** [comment]
Yes, sounds good.
Can you make another pull request for the name changes? Not sure if changing the addon name etc is valid and what needs changing then.

**Comment by infinity0** [comment]
I've pushed another commit to #27 which changes the repo URL in `install.rdf`. If you merge it, you should also :

- rename this repo to `mozilla-gnome-keyring`
- rename the [AMO page](https://addons.mozilla.org/en-US/firefox/addon/moz-gnome-keyring-integration-1) to `mozilla-gnome-keyring` (to match this repo's name) or to `gnome-keyring-integration` (to match the extension id).

We could also change the extension id in `Makefile` and `install.rdf` - but I thought that is actually unnecessary, and might make upgrading for existing users a bit more annoying - so I've left it as `gnome-keyring-integration`.

**Comment by infinity0** [comment]
Oh, and when you release, could you please tag the commit that corresponds to a particular version? For example it's unclear right now exactly which commit is 0.8 or 0.9.

**Comment by swick** [comment]
I think that's it.
